plugin/log/setup_test.go

```diff
@@ -17,56 +18,64 @@ func TestLogParse(t *testing.T) {
 		{`log`, false, []Rule{{
 			NameScope: ".",
 			Format:    DefaultLogFormat,
+			Class:     map[response.Class]bool{response.All: true},
```

Comment:
Can you just add a new test that checks the class stuff?

Reply:
ok